hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/ReconUtils.java

```diff
@@ -111,6 +109,43 @@ public class ReconUtils {
   public ReconUtils() {
   }
 
+  /**
+   * Get the current rebuild state of NSSummary tree.
+   * Delegates to NSSummaryTask's unified control mechanism.
+   * 
+   * @return current RebuildState from NSSummaryTask
+   */
+  public static org.apache.hadoop.ozone.recon.tasks.NSSummaryTask.RebuildState getNSSummaryRebuildState() {
+    return org.apache.hadoop.ozone.recon.tasks.NSSummaryTask.getRebuildState();
+  }
+
+  /**
+   * Convenience method to trigger asynchronous NSSummary tree rebuild.
+   * Uses the unified control mechanism in NSSummaryTask.
+   * 
+   * @param reconNamespaceSummaryManager The namespace summary manager
+   * @param omMetadataManager The OM metadata manager
+   * @return true if rebuild was triggered successfully, false otherwise
+   */
+  public static boolean triggerAsyncNSSummaryRebuild(
+      ReconNamespaceSummaryManager reconNamespaceSummaryManager,
+      ReconOMMetadataManager omMetadataManager) {
+    
+    // Submit rebuild task to single thread executor for async execution
+    NSSUMMARY_REBUILD_EXECUTOR.submit(() -> {
```

Comment:
Method returns boolean but doesn't use unified control state.
We can add the state to RUNNING state. 

```
public static boolean triggerAsyncNSSummaryRebuild(...) {
  RebuildState currentState = NSSummaryTask.getRebuildState();
  if (currentState == RebuildState.RUNNING) {
    log.info("NSSummary rebuild already in progress");
    return false;
  }
  
  NSSUMMARY_REBUILD_EXECUTOR.submit(() -> {
    // ... async work
  });
  return true;
}
```